watertap/property_models/tests/test_multicomp_aq_sol_prop_pack.py

```diff
@@ -2255,6 +2321,11 @@ def test_no_mw_data_provided():
         m.fs.properties = MCASParameterBlock(
             solute_list=["foo", "Na_+", "Cl_-"], ignore_neutral_charge=True
         )
+    # Choose capitalized solute names that won't be found in periodic table and omit mw_data
+    # Because of helper function logic, an example like following would result in an empty result
+    msg = "The symbol 'T' from the component name 'TDS' could not be found in the periodic table."
+    with pytest.raises(ConfigurationError, match=re.escape(msg)):
+        m.fs.mcas_props = MCASParameterBlock(solute_list=["TDS"])
```

Comment:
@adam-a-a It looks like this is currently causing test failures: 
![image](https://github.com/watertap-org/watertap/assets/48035537/1cf8ce74-7ac9-4e08-85cc-e585da463c6f)

I tried searching the code for that error message (i.e. the code that builds the string used in the error message), but I couldn't find anything. Do you have more information about what might be happening?